static/js/controllers/inbox.js

```diff
@@ -148,6 +148,16 @@ require('moment/locales');
         $scope.items = contacts || [];
       };
 
+      $scope.removeContact = function(contact) {
+        var i, id = contact._id;
+        for (i=$scope.items.length-1; i>=0; --i) {
+          if($scope.items[i]._id === id) {
+            $scope.items.splice(i, 1);
+            return;
+          }
+        }
+      };
+
```

Comment:
Consider using the underscore.js library here to make this more readable. _.reject or _.findIndex (if you upgrade to 1.8).